bundles/activity.rst

```diff
@@ -187,6 +187,39 @@ users with a ``view`` permission for the context:
         }
     }
 
+Extending the Admin View with a Activity Table
+------------------------------------------------
+
+To extend the admin view by adding a activities table, follow these steps:
+
+	1.	Inject the `ActivityViewBuilderFactoryInterface`: Inject this interface into your custom Admin class. This will allow you to utilize the necessary methods to create the activities view.
+	2.	Create the Activity List View: Use the `createActivityListViewBuilder` method to create the list view for the activities table.
+
+Here’s an example implementation from the `SnippetAdmin` class, demonstrating how to add the activities tab to your custom admin view:
+
+.. code-block:: php
+
+    if ($this->activityViewBuilderFactory->hasActivityListPermission()) {
+        $viewCollection->add(
+            $this->activityViewBuilderFactory
+                ->createActivityListViewBuilder(
+                    $insightsResourceTabViewName . '.activity',
+                    '/activities',
+                    SnippetDocument::RESOURCE_KEY
```

Comment:
Should we tell here to use your resource key CustomEntity::RESOURCE_KEY

Reply:
Adjusted it to `CustomEntity::RESOURCE_KEY`